rpm_spec/core-dom0-linux.spec.in

```diff
@@ -192,6 +192,28 @@ if [ $1 -ge 2 ]; then
     fi
 fi
 
+# setup thinpool autoextend
+if [ -e /etc/lvm/lvm.conf ]; then
+    tpool_threshold_line_value=$(grep -n -Po '^[^#]thin_pool_autoextend_threshold[ \t]*=[ \t]*\K\d+' /etc/lvm/lvm.conf)
```

Comment:
We can use `lvmconfig` here.

Reply:
Yes but as far as I've checked, I don't get line reference of the property that I need to insert comment.

Reply:
The idea I've in mind is that if we cannot predict properly where is located the property and its value in the file then don't try to modify anything even if `lvmconfig` would have gave the value (only?).